### ISSUE TITLE
Claiming in new beach biome throws error

### ISSUE THREAD

**Issue by Kraxxi** [GriefPrevention/GriefPrevention#319]
### What behaviour is observed:
Errors being thrown in console
### What behaviour is expected:
No errors.
### Steps/models to reproduce:
Generate New World
Find beach biome and claim an area inside or partially inside this biome.

### Spigot/Craftbukkit/Paper and GriefPrevention version:
Spigot: 1.13-R0.1-SNAPSHOT
GP: 16.8

### Stack trace/error

```
[09:05:23 WARN]: [GriefPrevention] Plugin GriefPrevention v16.8 generated an exception while executing task 32
java.lang.NoSuchFieldError: BEACHES
        at me.ryanhamshire.GriefPrevention.RestoreNatureProcessingTask.getPlayerBlocks(RestoreNatureProcessingTask.java:800) ~[?:?]
        at me.ryanhamshire.GriefPrevention.AutoExtendClaimTask.getLowestBuiltY(AutoExtendClaimTask.java:46) ~[?:?]
        at me.ryanhamshire.GriefPrevention.AutoExtendClaimTask.run(AutoExtendClaimTask.java:28) ~[?:?]
        at org.bukkit.craftbukkit.v1_13_R1.scheduler.CraftTask.run(CraftTask.java:76) ~[server.jar:git-Spigot-0b44fa0-38adf52]
        at org.bukkit.craftbukkit.v1_13_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52) [server.jar:git-Spigot-0b44fa0-38adf52]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_151]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_151]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_151]
```

### GriefPrevention config.yml (if applicable)

```yml
# Default values are perfect for most servers.  If you want to customize and have a question, look for the answer here first: http://dev.bukkit.org/bukkit-plugins/grief-prevention/pages/setup-and-configuration/
GriefPrevention:
  SeaLevelOverrides:
    world: -1
    world_nether: -1
    world_the_end: -1
  Claims:
    Mode:
      world: Survival
      world_nether: Disabled
      world_the_end: Disabled
    PreventGlobalMonsterEggs: true
    PreventTheft: true
    ProtectCreatures: true
    PreventButtonsSwitches: true
    LockWoodenDoors: false
    LockTrapDoors: false
    LockFenceGates: true
    EnderPearlsRequireAccessTrust: true
    ProtectHorses: true
    InitialBlocks: 100
    Claim Blocks Accrued Per Hour:
      Default: 100
    Max Accrued Claim Blocks:
      Default: 2000
    Accrued Idle Threshold: 0
    AccruedIdlePercent: 0
    AbandonReturnRatio: 1.0
    AutomaticNewPlayerClaimsRadius: 4
    ExtendIntoGroundDistance: 5
    MinimumWidth: 5
    MinimumArea: 100
    MaximumDepth: 0
    InvestigationTool: STICK
    ModificationTool: GOLD_SPADE
    Expiration:
      ChestClaimDays: 7
      UnusedClaimDays: 14
      AllClaims:
        DaysInactive: 60
        ExceptWhenOwnerHasTotalClaimBlocks: 10000
        ExceptWhenOwnerHasBonusClaimBlocks: 5000
      AutomaticNatureRestoration:
        SurvivalWorlds: false
    AllowTrappedInAdminClaims: false
    MaximumNumberOfClaimsPerPlayer: 0
    CreationRequiresWorldGuardBuildPermission: true
    PortalGenerationRequiresPermission: false
    VillagerTradingRequiresPermission: true
    CommandsRequiringAccessTrust: /sethome
    DeliverManuals: true
    ManualDeliveryDelaySeconds: 30
  Spam:
    Enabled: true
    LoginCooldownSeconds: 60
    LoginLogoutNotificationsPerMinute: 5
    ChatSlashCommands: /me;/global;/local
    WhisperSlashCommands: ''
    WarningMessage: Please reduce your noise level.
    BanOffenders: false
    BanMessage: Banned for spam.
    AllowedIpAddresses: 1.2.3.4; 5.6.7.8
    DeathMessageCooldownSeconds: 120
    Logout Message Delay In Seconds: 0
  PvP:
    RulesEnabledInWorld:
      world: true
      world_nether: true
      world_the_end: true
    ProtectFreshSpawns: true
    PunishLogout: true
    CombatTimeoutSeconds: 15
    AllowCombatItemDrop: false
    BlockedSlashCommands: /home;/vanish;/spawn;/tpa
    ProtectPlayersInLandClaims:
      PlayerOwnedClaims: true
      AdministrativeClaims: true
      AdministrativeSubdivisions: true
    AllowLavaDumpingNearOtherPlayers: true
    AllowFlintAndSteelNearOtherPlayers: true
    ProtectPetsOutsideLandClaims: false
  Economy:
    ClaimBlocksPurchaseCost: 0.0
    ClaimBlocksSellValue: 0.0
  ProtectItemsDroppedOnDeath:
    PvPWorlds: false
    NonPvPWorlds: true
  BlockLandClaimExplosions: true
  BlockSurfaceCreeperExplosions: true
  BlockSurfaceOtherExplosions: true
  LimitSkyTrees: true
  LimitTreeGrowth: false
  LimitPistonsToLandClaims: true
  FireSpreads: false
  FireDestroys: false
  AdminsGetWhispers: true
  AdminsGetSignNotifications: true
  SmartBan: true
  Mute New Players Using Banned Words: true
  MaxPlayersPerIpAddress: 3
  Siege:
    Worlds: []
    BreakableBlocks:
    - DIRT
    - GRASS
    - LONG_GRASS
    - COBBLESTONE
    - GRAVEL
    - SAND
    - GLASS
    - THIN_GLASS
    - WOOD
    - WOOL
    - SNOW
    DoorsOpenDelayInSeconds: 300
  EndermenMoveBlocks: false
  SilverfishBreakBlocks: false
  CreaturesTrampleCrops: false
  RabbitsEatCrops: true
  HardModeZombiesBreakDoors: false
  Database:
    URL: ''
    UserName: ''
    Password: ''
  UseBanCommand: false
  BanCommandPattern: ban %name% %reason%
  Mods:
    BlockIdsRequiringAccessTrust: []
    BlockIdsRequiringContainerTrust: []
    BlockIdsExplodable: []
    PlayersIgnoringAllClaims: []
  Advanced:
    fixNegativeClaimblockAmounts: true
  Abridged Logs:
    Days To Keep: 7
    Included Entry Types:
      Social Activity: true
      Suspicious Activity: true
      Administrative Activity: false
      Debug: false
      Muted Chat Messages: false

```

### Plugin list (if applicable):
Only GP installed

**Comment by RoboMWM** [comment]
I'm not aware of any way to resolve this other than pushing out a 1.13-compiled build. There's a PR in the pipeline, and if you're interested I could give you a preview build of it.

**Comment by NullCase** [comment]
(Also came to confirm this but i see it's not necessary.)

RoboMWM, i'm running a 1.13 test server and testing new builds on it.

It's online and running 24/7.

I'll run your new 1.13 version if you have something to test.

**Comment by RoboMWM** [comment]
Let me know if this is fixed in https://github.com/TechFortress/GriefPrevention/releases/tag/16.9.alpha

**Comment by NullCase** [comment]
It looks like this is fixed in 16.9 alpha.
